Server/src/routes/GraphsRouter.ts

```diff
@@ -10,20 +10,24 @@ import { JWTAuthenticator } from '../middleware/JWTAuthenticator';
 let router = Router();
 let savedGraphsControllerObject = new GraphsController();
 
-router.get('/api/v1/oneSavedGraph/:oneSavedGraph', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {
-    savedGraphsControllerObject.createRequestForSingleSavedGraph(request, response);
+router.get('/api/v1/graphStateAPI/:graphStateId', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {
+    savedGraphsControllerObject.createRequestForSingleGraph(request, response);
 });
 
-router.get('/api/v1/userSavedGraphs$', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {
+router.get('/api/v1/graphStateAPI$', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {
     savedGraphsControllerObject.createRequestForUserSavedGraphs(request, response);
 });
 
-router.post('/api/v1/addSavedGraph$', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {
-    savedGraphsControllerObject.createRequestForAddingSavedGraph(request, response);
+router.put('/api/v1/graphStateAPI$', JWTAuthenticator.verifyJWT, (request: Request, response: Response) => {

```

Comment:
its much easier for api consumer like frontend to deal with one route and different request types, good job on cleaning up the routes

Reply:
Yeah agreed this is much cleaner. We should keep them as 
/api/v1/{apiName}
Without putting api in there

Reply:
Will change